index.js

```diff
@@ -224,8 +224,11 @@ function parseMetadata(dataset){
     $('#meta-title').html(title);
     var responsiblePartiesString = "";
     for (let i = 0; i < responsibleParties.length; i++) {
-        responsiblePartiesString += responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:individualName"] + ", " + 
-                                    responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:role"]["gmd:CI_RoleCode"] + "<br/>"
+        role = responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:role"]["gmd:CI_RoleCode"];
+        if ( role === 'principalInvestigator' )
```

Comment:
Role also appears in line 218. So I'd suggest to add a method `formatRole` that does the formatting.
Plus, there are some more unreadable role texts (`pointOfContact` and lowercase `originator`/`distributor`), so the method would be something like:
```
function formatRole(role) {
    if (role === 'principalInvestor') {
        role = 'Principal Investor';
    } else if (role === 'pointOfContact') { 
        role = 'Point of Contact';
    }
    return role.charAt(0).toUpperCase() + string.slice(1);
}
```